html-css-popup-task

iva-stasia opened the pull request:
### Html-css-popup

[Demo](https://iva-stasia.github.io/html-css-popup/) |
[Code base](https://github.com/iva-stasia/html-css-popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

iva-stasia commented:
I checked the code for common mistakes and didn't find any.

A-Ostrovnyy commented on submissions/iva-stasia/html-css-popup/index.html:
```diff
@@ -0,0 +1,68 @@
+                <li class="p_item"><a href="#" class="p_icon apps"><span class="caption">Apps</span></a></li>
+                <li class="p_item"><a href="#" class="p_icon doc"><span class="caption">Docs</span></a></li>
+                <li class="details">
+                   <details>
+                    <summary class="more_button">More</summary><hr class="line">
```
:fire:

A-Ostrovnyy commented on submissions/iva-stasia/html-css-popup/index.html:
```diff
@@ -0,0 +1,68 @@
+          </ul>
+        </nav>
+      </header>
+      <main class="main">
+      </main>
```
Avoid unused tags in your code.

iva-stasia replied:
I used this tag to style and position the header and footer using flex. However, this is not necessary now.
Changed the way these elements were positioned without using flex and removed the empty main tag.

A-Ostrovnyy commented on submissions/iva-stasia/html-css-popup/index.html:
```diff
@@ -0,0 +1,68 @@
+      <main class="main">
+      </main>
+      <footer class="footer">
+        <div class="phrase">
+          Try to find something interesting here :)
+        </div>
```
Logically it looks like a link, what do you think?

iva-stasia replied:
Agree. Since I like the page with the footer, I decided to replace the phrase with a cat. Hopefully, it doesn't look like a button now :)

A-Ostrovnyy commented on submissions/iva-stasia/html-css-popup/styles.css:
```diff
@@ -0,0 +1,485 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600&display=swap');
```
:fire:

A-Ostrovnyy reviewed (changes requested):
Awesome design!
Sorry for late response

iva-stasia commented:
Made changes according to the comments.
I also found that my page did not work correctly on iOS devices: there were problems with checkboxes and outlines on icons. Therefore, I additionally changed the way the checkbox is hidden and removed the outlines for iOS.
I also added a small arrow at the top of the popup to visually "connect" the popup and the button from which it drops out.

iva-stasia commented:
Noticed that almost all of my html file looks like changed. This is because I removed the div with the wrapper class and moved the lines that were inside this tag.

A-Ostrovnyy reviewed (approved):
Awesome!
